.circleci/check_mysql_connection.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+[ -n "$MYSQL_USER" ] || MYSQL_USER=test
+[ -n "$MYSQL_PASSWORD" ] || MYSQL_PASSWORD=test
+
+[ -n "$NUM_PINGS" ] || NUM_PINGS=15
+[ -n "$DELAY" ] || DELAY=1
+
+for try in $(seq $NUM_PINGS); do
+    mysqladmin ping -h127.0.0.1 -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" --protocol=TCP && exit 0 || sleep $DELAY
+done
```

Comment:
Uh... I'd maybe make the database address a parameter as well. Wouldn't cost much here, in a separate file, but who knows when it might come useful. Otherwise this 127.0.0.1 sticks out like a sore thumb. (And I guess it can be `localhost`, to allow IPv6 where available.)